package-lock.json

```diff
@@ -7483,9 +7483,9 @@
       }
     },
     "node_modules/make-fetch-happen": {
-      "version": "13.0.0",
-      "resolved": "https://registry.npmjs.org/make-fetch-happen/-/make-fetch-happen-13.0.0.tgz",
-      "integrity": "sha512-7ThobcL8brtGo9CavByQrQi+23aIfgYU++wg4B87AIS8Rb2ZBt/MEaDqzA00Xwv/jUjAjYkLHjVolYuTLKda2A==",
+      "version": "13.0.1",
```

Comment:
Is it beneficial to update `make-fetch-happen`'s semver in package.json as well (via `install` instead of `update`)?

Reply:
Yes, good catch. I forgot this was a top level dependency.